src/MICore/CommandFactories/gdb.cs

```diff
@@ -201,6 +201,8 @@ public override Task ExecJump(string filename, int line)
 
         public override Task ExecJump(ulong address)
         {
+            return _debugger.ConsoleCmdAsync("set $pc = " + string.Format(CultureInfo.InvariantCulture, "0x{0:X}", address), false);
```

Comment:
The question is if some kind of error handling is needed.

Reply:
Ideally yes, but since this is a non-MI command, there aren't wonderful options for knowing it is succeeded. I am not sure if GDB supports assigning to `$pc` through MI commands instead (`-var-assign`?)

Reply:
Looks like it but requires a variable object incl. creation/deletion (possibly just once): https://sourceware.org/gdb/current/onlinedocs/gdb/GDB_002fMI-Variable-Objects.html